### PR TITLE
[Optional Reference]LPS-185546 module: dynamic-data-mapping-service

### PR DESCRIPTION
**Pull request by JanisZhang**
fix test. thanks!
previous: https://github.com/liferay-core-infra/liferay-portal/pull/4607
https://issues.liferay.com/browse/LPS-185546

### CODE REVIEW

**Comment by JanisZhang** [comment]
ci:test:startup

**Comment by Kvale1733** [comment]
ci:test:stable

**Comment by Kvale1733** [comment]
@JanisZhang No regressions found.

**Comment by dantewang** [comment]
@JanisZhang Conflicts...

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/test/java/com/liferay/dynamic/data/mapping/BaseDDMTestCase.java [hunk 1]
```diff
@@ -137,12 +138,17 @@ public static void setUpClass() {
...
+			bundleContext.registerService(
+				SiteConnectedGroupGroupProvider.class,
+				Mockito.mock(SiteConnectedGroupGroupProvider.class), null);
 	}
 
 	@AfterClass
 	public static void tearDownClass() {
 		_ddmFormDeserializerServiceRegistration.unregister();
-
+		_siteConnectedGroupGroupProviderServiceRegistration.unregister();
```
Comment (dantewang) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/test/java/com/liferay/dynamic/data/mapping/BaseDDMTestCase.java:151]: @JanisZhang I think we'd better add an empty line after this line, if you want to keep the mockedStatic.close last one in the method. Or Brian may ask you to sort in alphabet order...

**Comment by dantewang** [comment]
@JanisZhang Please resend, thanks!

**Pull request closed without merging**